layouts/css/base.styl

```diff
@@ -3,9 +3,11 @@
 // Base styles
 body
     box-sizing border-box
-    font 300 20px/1.5 'Source Sans Pro', Arial, sans-serif
-    color #000
+    font 400 15px/1.5 "source-sans-pro", sans-serif
```

Comment:
Hmmm, was there anything wrong with 20px as base size? Medium uses about 20px for most of everything, makes it easy to read?